manager/pom.xml

```diff
@@ -145,7 +145,7 @@
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <scope>provided</scope>
+<!--            <scope>provided</scope>-->
         </dependency>
```

Comment:
hi the mysql dependency must be `<scope>provided</scope>`

Due to the Apache Foundation's requirements for license compliance, HertzBeat's installation package cannot include dependencies with GPL licenses such as MySQL, Oracle, etc. Users need to add them themselves. Users can download the drivers from the following links and copy them to the ext-lib directory of the installation.

refer https://hertzbeat.apache.org/docs/help/mysql, suggest add help doc same with this.

Reply:
hi , it is my bad,  i tested on my local pc , and i forget to recover this dependency scope.